.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -1,8 +1,7 @@
-name: 🚀 功能请求 Feature Request
-description: 提出新的功能特性
+name: 🚀 功能增强 Enhancement
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)